experimental/oak_async/src/io.rs

```diff
@@ -100,4 +132,8 @@ impl<T: Decodable + Send> ReceiverAsync for oak::io::Receiver<T> {
     fn receive_async(&self) -> ChannelRead<Self::Message> {
         ChannelRead::new(self.handle)
     }
+
+    fn receive_stream(&self) -> ChannelReadStream<Self::Message> {
```

Comment:
Could this be a default implementation in `ReceiverAsync`?

Reply:
Yes! Thanks 😄